sql/moz-fx-data-marketing-prod/google_search_console_derived/search_impressions_by_site_v1/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+friendly_name: Search Impressions By Site
+description: |-
+  Google Search impressions aggregated by site, synced by Fivetran to BigQuery for the following domains:
+    * addons.mozilla.org
+    * blog.mozilla.org
+    * getpocket.com
+    * support.mozilla.org
+    * www.mozilla.org
+
+  Anonymized search queries aren't included.
+
+  We stopped syncing Google Search Console data with Fivetran in August 2023.
+  See https://bugzilla.mozilla.org/show_bug.cgi?id=1764960#c44.
+owners:
+- srose@mozilla.com
+labels:
+  incremental: true
+  owner1: srose
+scheduling:
```

Comment:
Is this not meant to be scheduled in a DAG or go into a default DAG?

Reply:
The v1 ETLs are intentionally not being scheduled.  They will just be run once/initialized and then sit there holding historical data.

Reply:
In that case should the scheduling key be removed or commented out along with a note why it is not scheduled?

Reply:
Sure, done. 👍